srcpkgs/python3-pyocr/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'python3-pyocr'
+pkgname=python3-pyocr
+version=0.8.5
+revision=1
+build_style=python3-module
+makedepends="python3-build python3-installer python3-setuptools_scm python3-wheel"
+depends="tesseract-ocr"
+short_desc="Python wrapper for Tesseract and Cuneiform"
+maintainer="idealist1508 <id1508-github@yahoo.com>"
+license="GPL-3.0-or-later"
+homepage="https://gitlab.gnome.org/World/OpenPaperwork/pyocr"
+distfiles="https://gitlab.gnome.org/World/OpenPaperwork/pyocr/-/archive/${version}/pyocr-${version}.tar.bz2"
+checksum=ba9c63cfac7bb6e441590053954466b41768e6b87f1b279c2d62460cad8bb174
+
+do_build() {
+	python -m build --wheel --no-isolation
```

Comment:
I have no idea why this line causes a build error on any architecture except x86_64 glibc and i686 glibc.

`/void-packages/srcpkgs/python3-pyocr/template: line 16: python: command not found
=> ERROR: python3-pyocr-0.8.5_1: do_build: 'python -m build --wheel --no-isolation' exited with 127
=> ERROR:   in do_build() at srcpkgs/python3-pyocr/template:16
`

Reply:
I found a solution.